validation/checks/itemSeq_check.go

```diff
@@ -48,6 +48,11 @@ func (check *ItemSeqCheck) Validate(profile string, location csv.Location, csvDa
 
 	value := csvData[location.RowIndex][location.ColIndex]
 
+	// value of "Item Sequence" is allowed to be null
+	if value == "" {
```

Comment:
Per the question and answer on the ticket, we want to only allow empty values for resources whose `Object Type` isn't `Page`. Page still must have an integer value.